app/views/result_filters/location/_form.html.erb

```diff
@@ -12,7 +12,7 @@
           "provider-error"
         end
     } %>
-    <%= form_with url: location_path, method: :post do |form| %>
+    <%= form_with url: location_path, method: :post, data: { "ga-event-form" => "Location" } do |form| %>
```

Comment:
If we going to add this then we should add it for other forms too:

```Funding filter form data-ga-event-form="Salary"
Location filter form data-ga-event-form="Location"
Location filter form data-ga-event-form="Provider"
Qualification filter form data-ga-event-form="Qualification"
StudyType filter form data-ga-event-form="Study Type"
Subject filter form data-ga-event-form="Subject"
Vacancy filter form data-ga-event-form="Vacancy"```

Reply:
@aliuk2012 yeah, ive just picked up that card

Reply:
👍 I was just addressing the cookie pr comments first otherwise I would have just submitted a commit to cover those other usages